src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
+
+  const handleClickSearch = (e) => {
+    window.open(`products/${searchItem}`)
+    setSearch("")
+  }
+
+  const handleKeyPress = (e) => {
+    if (e.key === 'Enter') {
```

Comment:
Similar to this [comment](https://github.com/Cobda/cobda-web/pull/38/files#r584076437)
 
```suggestion
    if (e.key === 'Enter' && searchItem.trim()) {
 {
```